### ISSUE TITLE
[F-Req] Mark PojavLauncher as a game

### ISSUE THREAD

**Issue by ghost** [PojavLauncherTeam/PojavLauncher#5929]
### Is this request related to a bug?

No

### Describe the request.

Optimizations by OEMs for gaming aren't applied and apps by OEMs for gaming cannot be used because PojavLauncher is marked as an app, not a game. Mark PojavLauncher as a game instead of an app so optimizations are applied and apps by OEMs for gaming can be used.

**Comment by ghost** [comment]
Same. In game turbo (it's system app on any xiaomi phones. Game turbo is in security app) on my xiaomi phone pojav is not applied as game I can add it in games but graphical changes in game turbo is not applie for pojav or doesn't change anything but in genshin impact or call of duty affect. I know pojav launcher is game utility but run game so add this is good for it

**Comment by kasrarouhi** [comment]
I meaning about this settings
![Screenshot_2024-08-15-18-57-30-431_com miui securityadd](https://github.com/user-attachments/assets/2a72f9ad-d2c6-4866-a2ff-546b7df5aefd)

**Comment by hiimabignoobwihenolive** [comment]
I have a honor x6 and it detects the gplay version as a game but not the GitHub release it would be very helpful if it applied for the GitHub version

**Comment by kasrarouhi** [comment]
> I have a honor x6 and it detects the gplay version as a game but not the GitHub release it would be very helpful if it applied for the GitHub version

In xiaomi game turbo it's not detect pojav launcher play store version as a game for me

**Comment by ghost** [comment]
In xiaomi game turbo didn't detect pojav launcher play store version as a game for me

https://github.com/user-attachments/assets/89604989-59f8-4a93-9fe5-48f325acba3a



Is no any notification about pojav add to game turbo or add in game turbo. Usually game turbo add games automaticly and send notification about add this but not for pojav

**Comment by lzfddkd** [comment]
You want Xiaomi Game Turbo send a notification like this when PojavLauncher has been installed,yes?
![Screenshot_2024-09-16-23-05-50-661_com peopledailychina activity](https://github.com/user-attachments/assets/4ffbce60-65d5-459e-8662-a4112b149003)

**Comment by lzfddkd** [comment]
Let Xiaomi Game Turbo think PojavLauncher is a game,just like Minecraft (Bedrock Edition on Android),This is what you want,right?
![Screenshot_2024-09-16-23-15-59-004_com miui securityadd](https://github.com/user-attachments/assets/083f2bc9-622c-485d-bf94-fd112f7e0ca9)

**Comment by ghost** [comment]
Yes exactly. And for more compatible with game turbo settings. Many graphical settings doesn't affect on pojav but other games do

**Comment by Jokypond** [comment]
This could be easy fixed by doing something like [this](https://gist.github.com/Adnan-Bacic/718eb3b4e70380696c91dc21c4804112), but asking to place it as a game instead of asking to remove it instead. Of course, this would only work on Samsung devices so you would need to see how other phone manufacturers deal with this.
